src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -184,10 +184,12 @@ public override FormulaValue Visit(BinaryOpNode node, SymbolContext context)
                 case BinaryOpKind.EqMedia:
                 case BinaryOpKind.EqBlob:
                 case BinaryOpKind.EqGuid:
+                case BinaryOpKind.EqOptionSetValue:
                     return OperatorBinaryEq(this, context, node.IRContext, args);
```

Comment:
out of curiosity, what's the reason for sending args instead of arg1, arg2 across all binary ops?

Reply:
So that these impls can re-use the same error handling pipeline for functions. 

``` 
        public static FunctionPtr OperatorBinaryNeq = StandardErrorHandling<FormulaValue>(
            expandArguments: NoArgExpansion,
            replaceBlankValues: DoNotReplaceBlank,
            checkRuntimeTypes: DeferRuntimeTypeChecking,
            checkRuntimeValues: DeferRuntimeValueChecking,
            returnBehavior: ReturnBehavior.AlwaysEvaluateAndReturnResult,
            targetFunction: NotEqual);
````